### PR TITLE
Add student's pin to the map

### LINKED ISSUE #11244: Add me!
Hello! :wave: Please add me as a collaborator to this repo.

### PR DESCRIPTION
**Pull request by Ahmadkhan12345566**
adding ak to map
closes  #11244

### CODE REVIEW

**Comment by raoasifraza11** [comment]
You may have trouble please visit https://vimeo.com/201011288

**Comment by Ahmadkhan12345566** [comment]
@raoasifraza11 
i follow the steps in the video   but again failed

**Comment by raoasifraza11** [comment]
Look like an issue of you first commet 🥇

**Comment by Ahmadkhan12345566** [comment]
ok

**Comment by raoasifraza11** [comment]
Dear you had missing the clossing end """   missing: `./_pins/githubstudentbita.json`.

File: _pins/githubstudentbita.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 ---
-githubHandle:Ahmadkhan12345566
-latitude:33.7294° N
-longitude:73.0931° E
+githubHandle: Ahmadkhan12345566
+latitude: 33.7294
+longitude: 73.0931
```
Comment (raoasifraza11) [inline _pins/githubstudentbita.json]: here you must close the start with ---

**Pull request merged**